eox_nelp/notifications/tests/test_models.py

```diff
@@ -14,8 +15,8 @@
 
 class UpcomingCourseDueDateTestCase(unittest.TestCase):
     """Test class for model UpcomingCourseDueDate"""
-
-    def test_nofity_log(self):
+    @patch("eox_nelp.notifications.tasks.notify_upcoming_course_due_date_by_id")
+    def test_nofity_log(self, *args):  # pylint: disable=unused-argument
```

Comment:
I think we should improve this test and rename it to test `test_send_notifications` and verify that the task was called